builder/builder-next/worker/worker.go

```diff
@@ -186,13 +188,13 @@ func (w *Worker) Close() error {
 	return nil
 }
 
-// ContentStore returns content store
-func (w *Worker) ContentStore() content.Store {
+// ContentStore returns the wrapped content store
+func (w *Worker) ContentStore() *containerdsnapshot.Store {
```

Comment:
Why is this changed from interface to the pointer type? I couldn't find where it was used differently in this PR